tests/Option.Tests.Unit/OptionExTests.cs

```diff
@@ -1,55 +1,58 @@
-﻿
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+﻿using Option.Tests.Unit.Helpers;
+using Xunit;
 
 namespace Option.Tests.Unit
-{
-    [TestClass]
+{
+    
     public class OptionExTests
-    {
-        [TestMethod]
-        public void OptionExToNullable()
+    {
+
+        #region OptionExtensions ToNullable
+        [Fact]
+        public void OptionExtensions_ToNullableCalledOnOptionContainingObject_ObjectShouldBeRetained()
         {
             var counter = Option.Some(new Counter()).ToNullable();
 
-            Assert.IsTrue(counter.HasValue);
-            Assert.AreEqual(0, counter.Value.Item);
-            Assert.AreEqual(0, counter.Value.Count);
+            Assert.True(counter.HasValue);
+            Assert.Equal(0, counter.Value.Item);
+            Assert.Equal(0, counter.Value.Count);
         }
 
-        [TestMethod]
-        public void OptionExToNullableWithItems()
+        
+        [Fact]
+        public void OptionExtensions_ToNullableCalledOnOptionContainingObject_StateOfObjectShouldBeRetained()
         {
             var counter = Option.Some(new Counter{Count = 1, Item = 2}).ToNullable();
 
-            Assert.IsTrue(counter.HasValue);
-            Assert.AreEqual(2, counter.Value.Item);
-            Assert.AreEqual(1, counter.Value.Count);
-        }
-
-        [TestMethod]
-        public void OptionExSelectSome()
+            Assert.True(counter.HasValue);
+            Assert.Equal(2, counter.Value.Item);
+            Assert.Equal(1, counter.Value.Count);
+        }
+
+        #endregion
+
+        #region OptionExtensions Select
+
+        [Fact]
+        public void OptionExtensions_SelectSomeCalledWithLambda_ValueAndResultOfLambdaShouldBeEquals()
```

Comment:
Not a big deal but function name ends with "Equals", "Equal" would be more appropriate.